transport/src/main/java/io/netty/channel/AbstractCoalescingBufferQueue.java

```diff
@@ -230,15 +230,19 @@ public final void writeAndRemoveAll(ChannelHandlerContext ctx) {
                 if (entry == null) {
                     if (previousBuf != null) {
                         decrementReadableBytes(previousBuf.readableBytes());
-                        ctx.write(previousBuf, ctx.voidPromise());
+                        // Mimic what voidPromise() did before.
```

Comment:
I think it would be better if the comment described the effect, and why it's desirable. This PR wishes to make voidPromise a thing of the past, so it's not a good example for describing new code.